Library/Homebrew/os/linux.rb

```diff
@@ -63,6 +63,8 @@ def sdk_path
       nil
     end
 
+    def symlink_current_pkgconfig_directory; end
```

Comment:
@MikeMcQuaid I know you don't like these `MacOS` compat methods, so I can move this up into the `OS` module if preferred. It kinda makes sense for it to be here for me, though.